ObedZac/iREPORTER
# Ch setting up database 162470331

# 162470331 Starts setting up the data base and registers version two of app

### What does this PR do?

This PR sets up my data base and registers version 2 of the app

### Description of Task to be completed?

•Register version two of the app
• Setup a data base called ireporter.

### How should this be manually tested?

Test the endpoints on postman to see if version two is working by testing the registered urls in version two. Look at db_con.py to see that data base has been set up correctly.
### What are the relevant pivotal tracker stories?

162470331
